temporal-normalizer/src/test/java/edu/illinois/cs/cogcomp/temporal/normalizer/tests/TestTemporalChunker.java

```diff
@@ -0,0 +1,354 @@
+        AnnotatorService pipeline = PipelineFactory.buildPipeline(new ResourceManager(props));
+
+        System.out.println("Working Directory = " +
+                System.getProperty("user.dir"));
+        ResourceManager nerRm = new TemporalChunkerConfigurator().getDefaultConfig();
+        IOUtilities.existsInClasspath(TemporalChunkerAnnotator.class, nerRm.getString("modelDirPath"));
+
+        java.util.logging.Logger.getLogger("HeidelTimeStandalone").setLevel(Level.OFF);
+
+        long preprocessTime = System.currentTimeMillis();
+        List <TextAnnotation> taList = new ArrayList<>();
+        for (int j = 0; j < te3inputText.size(); j ++) {
+            String filename = "./text/"+docIDs.get(j);
+            PrintStream ps = new PrintStream(filename);
+            ps.print(testText.get(j));
+            ps.close();
+
+            FileInputStream fis = new FileInputStream(filename);
+            int content;
+            String text = "";
+            while ((content = fis.read()) != -1) {
+                // convert to char and display it
+                text += (char) content;
+            }
+
+            text = text.replaceAll("\\n", " ");
+            TextAnnotation ta = pipeline.createAnnotatedTextAnnotation( "corpus", "id",  text);
+            taList.add(ta);
+        }
+
+        long startTime = System.currentTimeMillis();
+
+        int numTimex = 0;
+
+        String outputFolder = "AQ_goldext_illininorm";
+        File outDir = new File(outputFolder);
+        if (!outDir.exists()) {
+            outDir.mkdir();
+        }
+
+        for (int j = 0; j < te3inputText.size(); j ++) {
+//            if(!docIDs.get(j).contains("CNN_20130322_1003")) {
+//                continue;
+//            }
+
+            TextAnnotation ta = taList.get(j);
+            tca.addDocumentCreationTime(DCTs.get(j));
+
+            System.out.println(docIDs.get(j));
+
+
+            try {
+                List<TimexChunk> timex = tca.extractTimexFromFile(te3inputText.get(j), testText.get(j), ta);
+//                for (TimexChunk tc:timex)
+//                    System.out.println(tc.toTIMEXString());
+                tca.setTimex(timex);
+                String outputFileName = outputFolder + "/" +docIDs.get(j) + ".tml";
+                //String outputFileName = "AQ_goldext_htnorm/" +docIDs.get(j) + ".tml";
+
+                tca.write2Text(outputFileName, docIDs.get(j) ,testText.get(j));
+                numTimex += timex.size();
+                tca.deleteTimex();
+            } catch (AnnotatorException e) {
+                fail("Exception while adding TIMEX3 VIEW " + e.getStackTrace());
+            }
+
+
+
+        }
+        long endTime   = System.currentTimeMillis();
+        long totalTime = endTime - startTime;
+        System.out.println("Process time: " + totalTime);
+        System.out.println("Preprocess + process time: " + (endTime-preprocessTime) );
+        System.out.println("Total timex3: " + numTimex );
```

Comment:
What about evaluating performance of the system in terms of precision/recall/F1, or whatever other metrics are the standard in this task?